<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>digitalClass</class>
 <widget class="QMainWindow" name="digitalClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>digital</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>600</width>
      <height>550</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>550</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>550</height>
     </size>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>201</height>
      </rect>
     </property>
     <attribute name="label">
      <string>网格信息</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>顶点数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ver_num">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>面数:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_face_num">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>边数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_edge_num">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>201</height>
      </rect>
     </property>
     <attribute name="label">
      <string>绘制图形</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>77</width>
        <height>83</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_default_color">
         <property name="text">
          <string>默认颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_consec_color">
         <property name="text">
          <string>连续数值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_discrete_color">
         <property name="text">
          <string>离散数值</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>201</height>
      </rect>
     </property>
     <attribute name="label">
      <string>绘制相邻面</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>164</width>
        <height>147</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>点id：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spin_pointid">
               <property name="maximum">
                <number>9999999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_point">
               <property name="text">
                <string>相邻点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_pt_face">
               <property name="text">
                <string>相邻面</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>面id：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_faceid"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_normal">
           <property name="text">
            <string>法向量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_face_face">
           <property name="text">
            <string>相邻面</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clear">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>201</height>
      </rect>
     </property>
     <attribute name="label">
      <string>绘制区域</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>31</y>
        <width>160</width>
        <height>85</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="btn_open_data">
         <property name="text">
          <string>打开输入文件</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btn_color_two">
           <property name="text">
            <string>两点染色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_color_three">
           <property name="text">
            <string>三点染色</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clr_region">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>201</height>
      </rect>
     </property>
     <attribute name="label">
      <string>ICP</string>
     </attribute>
     <widget class="QLabel" name="label_output">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>158</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btn_open_model2">
         <property name="text">
          <string>打开模型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_icp">
         <property name="text">
          <string>ICP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="digital.qrc"/>
 </resources>
 <connections/>
</ui>
